init.rb

```diff
@@ -11,30 +11,29 @@ def prompt(message, default)
 
 folder_path = __dir__
 
-default_package_name = 'MyPackage'
+default_package_name = File.basename(Dir.getwd)
 default_bundle_domain = 'no.hyper'
-default_author_name = 'Hyper Interaktiv AS'
-default_author_email = 'ios@hyper.no'
-default_username = 'hyperoslo'
+default_author_name = `git config user.name`.strip
+default_author_email = `git config user.email`.strip
+default_username = default_author_email.split('@').first
```

Comment:
If I use this now then it will use my personal email and show me as the author :/

Reply:
Would it be better to use an env variable here?

Reply:
@zenangst I recognise some others are using this too. Just want to make it a bit. Or we could fetch these information from `cocoapods trunk me`?

Reply:
Yeah I totally get that :) I'm just curious how we could fix this for everyone :)